screwdriver.yaml

```diff
@@ -9,17 +9,16 @@ jobs:
     main:
         requires: [~pr, ~commit]
         steps:
-            - get: go get -v ./...
+            - modverify: go mod verify
             - vet: go vet ./...
-            - gofmt: "find . -name '*.go' | xargs gofmt -s -w"
+            - gofmt: gofmt -d .
```

Comment:
Thanks for fixing this step. Like other Screwdriver.cd go components, this `gofmt` step is useless 😥
https://github.com/screwdriver-cd/launcher/blob/862c0429331cf6fba21bdf97cb15e2950025d63c/screwdriver.yaml#L17

But `gofmt -d` returns always `0` even if there are any diffs.

```
# remove an empty line of meta.go
$ gofmt -d .
diff -u meta.go.orig meta.go
--- meta.go.orig	2019-07-16 21:42:50.000000000 +0900
+++ meta.go	2019-07-16 21:42:50.000000000 +0900
@@ -1,4 +1,5 @@
 package main
+
 import (
 	"bytes"
 	"encoding/json"

$ echo $?
0
```

How about using `(! gofmt -s -d . | grep '^')`?
It returns `1` when `gofmt` is necessary.
```
$ (! gofmt -s -d . | grep '^')
diff -u meta.go.orig meta.go
--- meta.go.orig	2019-07-16 21:51:21.000000000 +0900
+++ meta.go	2019-07-16 21:51:21.000000000 +0900
@@ -1,4 +1,5 @@
 package main
+
 import (
 	"bytes"
 	"encoding/json"
$ echo $?
1
```

Then there is no need to fix, it returns `0`.
```
$ git checkout -- meta.go
$ (! gofmt -s -d . | grep '^')
$ echo $?
0
```

Reply:
you are right, `gofmt` never returns an error code - https://github.com/golang/go/issues/24230.

`-s` flag (simplify code) is probably useless in the context of CI build and may even make it more difficult to map the simplified code to the original when trying to see where the formatting deviation is.

I'll change the PR to use `(! gofmt -d . | grep '^')`, ok?